docs/core/howto/ssl.rst

```diff
@@ -204,10 +199,40 @@ For example,
     options = CertificateOptions(..., dhParameters=dhParams)
 
 Another part of the TLS protocol which ``CertificateOptions`` can control is the version of the TLS or SSL protocol used.
-This is often called the context's "method".
+There are two ways to configure it, either by configuring a list of protocols, or by the legacy way of setting the internal context's "method".
+By default, Twisted will configure it to use TLSv1.0 or later and disable the insecure SSLv3 protocol, but this manual control over protocols can be helpful if you need to support legacy SSLv3 systems, or you wish to restrict it down to just the strongest of the TLS versions.
```

Comment:
I'm confused about this feedback.  What's wrong with the commas?  One of them, at least, "By default, Twisted", seems to serve the purpose of clarifying the default behavior.  "By default Twisted" might mean "With the default (version of) Twisted"; this is of course made clear later in the sentence but requiring the reader to move forward in order to understand the context just makes the sentence harder to read.

Similarly, it is common practice to separate independent clauses with a comma and a conjunction.  See rule 3b here: http://www.grammarbook.com/punctuation/commas.asp

Rule 3c on that page implies implies that the comma _is_ necessary in this case, since the independent clause after "but" has its own subject.

If I were going to have feedback on this sentence it would be "too many independent clauses in one sentence" :).